src/test/java/api/requests/scenarios/LoanDueDatesAfterRecallTests.java

```diff
@@ -845,4 +846,57 @@ public void shouldNotExtendLoanDueDateIfOverdueLoanIsRecalled() {
       hasJsonPath("loan.itemStatus", "Checked out"))
     ));
   }
+
+  @Test
+  public void loanDueDateTruncatedOnCheckoutWhenRecallAnywhereInQueue() {
+    final IndividualResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet();
+    final IndividualResource requestServicePoint = servicePointsFixture.cd1();
+    final IndividualResource jessica = usersFixture.jessica();
+    final IndividualResource charlotte = usersFixture.charlotte();
+    final IndividualResource james = usersFixture.james();
+    final IndividualResource rebecca = usersFixture.rebecca();
+    final IndividualResource steve = usersFixture.steve();
+
+    final Period loanPeriod = Period.weeks(4);
+    setFallbackPolicies(new LoanPolicyBuilder()
+      .withName("Can Circulate Rolling With Recalls")
+      .withDescription("Can circulate item With Recalls")
+      .rolling(loanPeriod)
+      .unlimitedRenewals()
+      .renewFromSystemDate()
+      .withRecallsMinimumGuaranteedLoanPeriod(Period.weeks(2))
+      .withRecallsRecallReturnInterval(Period.weeks(2)));
+
+      checkOutFixture.checkOutByBarcode(
+        smallAngryPlanet, steve, now(UTC));
```

Comment:
Could this fit on a single line?

```suggestion
      checkOutFixture.checkOutByBarcode(smallAngryPlanet, steve, now(UTC));
```